addons/l10n_ar/models/account_move.py

```diff
@@ -341,12 +343,19 @@ def _l10n_ar_get_invoice_totals_for_report(self):
                 if not x['tax_repartition_line'].tax_id.tax_group_id.l10n_ar_vat_afip_code
             ]
 
-        return self.env['account.tax']._prepare_tax_totals(
+        tax_totals = self.env['account.tax']._prepare_tax_totals(
             base_line_vals_list,
             self.currency_id,
             tax_lines=tax_line_vals_list,
         )
 
+        if include_vat:
+            temp = self.tax_totals
+            tax_totals['amount_total'] = temp['amount_total']
+            tax_totals['formatted_amount_total'] = temp['formatted_amount_total']
```

Comment:
we do an entire method an then you need to manipulate the results with an existing field?  Something is off here.